x-pack/plugins/data_enhanced/common/search/types.ts

```diff
@@ -15,20 +15,6 @@ import {
 
 export const ENHANCED_ES_SEARCH_STRATEGY = 'ese';
 
-export interface IAsyncSearchRequest extends IEsSearchRequest {
-  /**
-   * The ID received from the response from the initial request
-   */
-  id?: string;
-}
-
-export interface IEnhancedEsSearchRequest extends IEsSearchRequest {
```

Comment:
The `isRollup` parameter was replaced by the `indexType` parameter in the base `IEsSearchRequest` class, so this class is also not needed.